repository-containers/github.com/tensorflow/addons/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM tfaddons/dev_container:latest
+
+# This Dockerfile adds a non-root user with sudo access. Use the "remoteUser"
+# property in devcontainer.json to use it. On Linux, the container user's GID/UIDs
+# will be updated to match your local UID/GID (when using the dockerFile property).
+# See https://aka.ms/vscode-remote/containers/non-root-user for details.
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Options for common package install script - SHA updated on release
+ARG INSTALL_ZSH="false"
+ARG UPGRADE_PACKAGES="true"
```

Comment:
@seanpmorgan Do we want to update the package by default or not?

Reply:
I'd prefer they not be updated in this Dockerfile but rather done in tfaddons/dev_container if needed